src/Basic Example After CZML Fix.ipynb

```diff
@@ -0,0 +1,183 @@
+       "Full-screen buttons need human intervention to be triggered.***/\n",
+       "function openFullscreen(elem) {\n",
+       "console.log('Open Full Screen Called');\n",
+       "  if (elem.requestFullscreen) {\n",
+       "  console.log('condition1', elem.id);\n",
+       "    elem.webkitRequestFullScreen();\n",
+       "  } else if (elem.webkitRequestFullscreen) { \n",
+       "  console.log('condition2', elem.id);\n",
+       "    elem.webkitRequestFullScreen();\n",
+       "  } else if (elem.msRequestFullscreen) { \n",
+       "  console.log('condition3', elem.id);\n",
+       "    elem.msRequestFullscreen();\n",
+       "  }\n",
+       "}\n",
+       "\n",
+       "widget_element = document.getElementById(\"cesiumContainer-<function uuid4 at 0x0000019ABA7851F8>\");\n",
+       "button_class = \"cesium-button cesium-fullscreenButton\";\n",
+       "\n",
+       "setTimeout(function(){\n",
+       "\n",
+       "widget_element.getElementsByClassName(button_class)[0].addEventListener('click',function(){openFullscreen(document.getElementById(\"cesiumContainer-<function uuid4 at 0x0000019ABA7851F8>\"));});\n",
+       "console.log(widget_element.getElementsByClassName(button_class)[0]);\n",
+       "}, 150);\n",
+       "/*** Custom Script for binding the Full Screen button ends ***/\n",
+       "</script>\n"
+      ],
+      "text/plain": [
+       "<czml3.widget.CZMLWidget at 0x19abb4638c8>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "\n",
+    "from czml3.examples import simple\n",
+    "from czml3.widget import CZMLWidget\n",
+    "example_widget = CZMLWidget(\n",
+    "               document = simple, \n",
+    "               cesium_version = '1.79.1', \n",
+    "               ion_token=\"<Your Cesium Ion token. Obtain a token for free at https://cesium.com/ion/token\", \n",
+    "               terrain = \"Cesium.createWorldTerrain()\", \n",
+    "               imagery=\"Cesium.createWorldImagery()\", \n",
+    "               container_id = \"example_widget\"\n",
+    "          )\n",
+    "example_widget"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "seeing-lodging",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "\n",
+       "        <script >\n",
+       "        console.log(\"nihars_custom_plot\");\n",
+       "            document.getElementById('cesiumContainer-nihars_custom_plot').requestFullscreen();\n",
+       "        </script>\n",
+       "        "
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# triggers full-screen of the above widget\n",
+    "example_widget.request_full_screen()"
```

Comment:
The need to call an extra method is very troublesome to me.

Reply:
Indeed. It's just a temporary fix. I am actively exploring the root cause of this problem and ways to eliminate it. The fullscreen itself is working but the JavaScript that comes with Cesium Assets is not able to detect the right container to expand into to fullscreen.